daemon/indexing/lib/search.js

```diff
@@ -212,9 +216,11 @@ class Listing {
 
     const maxPrice = aggregationResponse.aggregations.max_price.value
     const minPrice = aggregationResponse.aggregations.min_price.value
+    const listings_total = searchResponse.hits.total
```

Comment:
super nit: naming is hard. maybe "totalNumListings" would be slightly more indicative for this variable ?

Reply:
yup agreed